### PR TITLE
adding a note on Negative Index

### PR DESCRIPTION
**Pull request by shouldsee**
#81 
I am not good at writing .rst but here is my attempt.

### CODE REVIEW

File: python2/doc/genomic.rst [hunk 1]
```diff
@@ -361,6 +361,33 @@ one object. The __iadd__ method is overloaded to add elements to the sets:
 
    Instantiation is as in :class:`GenomicArray`, only that ``datatype`` is always ``'O'``.
    
+
+Negative Index
+========
+
+Sometimes a negative index is useful for representing upstream elements. Imagine you would like to pool information about [-500,500) near transcription start site, it would be useful to have a such :class:`GenomicArray`. But unfortunately the current implementation means a such naive approach would raise an error, in
+
```
Comment (iosonofabio) [inline python2/doc/genomic.rst]: Thanks for this PR. Please remove the part about naive approaches and the error. We should tell them what to do, not what to NOT do ;-)

Comment (shouldsee) [reply python2/doc/genomic.rst]: Feel free to edit. I just find it helpful to record the whole process, just so if people can find this solution if they have a similar error.

Comment (iosonofabio) [reply python2/doc/genomic.rst]: Ok, I will edit as soon as I find some time.

Comment (shouldsee) [reply python2/doc/genomic.rst]: And I feel it justified to contrast two approaches here, even if one of them is not working

Comment (iosonofabio) [reply python2/doc/genomic.rst]: Yeah I don't feel the same, but thank you, I'll edit and merge.

**Comment by iosonofabio** [comment]
I added this with a more positive wording in commit cb71c56, closing.

**Pull request closed without merging**